[2025-05-21T16:20:14.465+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T16:20:14.492+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-05-21T16:20:14.505+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-05-21T16:20:14.506+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 6
[2025-05-21T16:20:14.539+0000] {taskinstance.py:2907} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2025-01-30 00:00:00+00:00
[2025-05-21T16:20:14.550+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=11066) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-21T16:20:14.551+0000] {standard_task_runner.py:72} INFO - Started process 11082 to run task
[2025-05-21T16:20:14.555+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2025-01-30T00:00:00+00:00', '--job-id', '3580', '--raw', '--subdir', 'DAGS_FOLDER/dar_with_taskflow_api.py', '--cfg-path', '/tmp/tmpfcqv1wd9']
[2025-05-21T16:20:14.557+0000] {standard_task_runner.py:105} INFO - Job 3580: Subtask get_name
[2025-05-21T16:20:14.639+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-01-30T00:00:00+00:00 [running]> on host 2032228d86df
[2025-05-21T16:20:15.073+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-30T00:00:00+00:00'
[2025-05-21T16:20:15.076+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-05-21T16:20:15.085+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2025-05-21T16:20:15.187+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-05-21T16:20:15.188+0000] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, run_id=scheduled__2025-01-30T00:00:00+00:00, execution_date=20250130T000000, start_date=20250521T162014, end_date=20250521T162015
[2025-05-21T16:20:15.291+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T16:20:15.393+0000] {taskinstance.py:3924} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-21T16:20:15.408+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
